server/server.ts

```diff
@@ -28,9 +29,15 @@ app.get('/api/hello', (req, res) => {
   res.json({ message: 'Hello, World!' });
 });
 
+// Property Routes
 app.use('/api/properties', propertyRoutes);
+
```

Comment:
I believe that you can require authorization on all those routes by changing this to
`app.use('/api/properties', authMiddleware, propertyRoutes);`
Then you won't need to explicitly add the auth middleware to the routes in propertyRoutes.